flask_admin/translations/af/LC_MESSAGES/admin.po

```diff
@@ -368,274 +358,242 @@ msgstr "Integriteit fout. %(message)s"
 
-#: ../flask_admin/templates/bootstrap2/admin/model/layout.html:59
-#: ../flask_admin/templates/bootstrap2/admin/model/layout.html:66
-#: ../flask_admin/templates/bootstrap3/admin/model/layout.html:59
-#: ../flask_admin/templates/bootstrap3/admin/model/layout.html:64
+#: ../flask_admin/templates/bootstrap4/admin/model/layout.html:80
+#: ../flask_admin/templates/bootstrap4/admin/model/layout.html:93
+#, python-format
+msgid "%(placeholder)s"
+msgstr ""
+
+#: ../flask_admin/templates/bootstrap4/admin/model/layout.html:88
+#: ../flask_admin/templates/bootstrap4/admin/model/layout.html:94
 msgid "Search"
 msgstr "Soek"
 
-#: ../flask_admin/templates/bootstrap2/admin/model/layout.html:74
-#: ../flask_admin/templates/bootstrap2/admin/model/layout.html:77
-#: ../flask_admin/templates/bootstrap2/admin/model/layout.html:78
-#: ../flask_admin/templates/bootstrap2/admin/model/layout.html:79
-#: ../flask_admin/templates/bootstrap3/admin/model/layout.html:72
-#: ../flask_admin/templates/bootstrap3/admin/model/layout.html:75
-#: ../flask_admin/templates/bootstrap3/admin/model/layout.html:76
-#: ../flask_admin/templates/bootstrap3/admin/model/layout.html:77
+#: ../flask_admin/templates/bootstrap4/admin/model/layout.html:102
 msgid "items"
 msgstr ""
 
-#: ../flask_admin/templates/bootstrap2/admin/model/list.html:23
-#: ../flask_admin/templates/bootstrap2/admin/model/list.html:25
-#: ../flask_admin/templates/bootstrap2/admin/model/modals/create.html:22
-#: ../flask_admin/templates/bootstrap3/admin/model/list.html:23
-#: ../flask_admin/templates/bootstrap3/admin/model/list.html:25
-#: ../flask_admin/templates/bootstrap3/admin/model/modals/create.html:10
+#: ../flask_admin/templates/bootstrap4/admin/model/list.html:23
+#: ../flask_admin/templates/bootstrap4/admin/model/list.html:25
+#: ../flask_admin/templates/bootstrap4/admin/model/modals/create.html:10
 msgid "Create New Record"
 msgstr "Skep Nuwe Rekord"
 
-#: ../flask_admin/templates/bootstrap2/admin/model/list.html:77
-#: ../flask_admin/templates/bootstrap3/admin/model/list.html:76
+#: ../flask_admin/templates/bootstrap4/admin/model/list.html:76
 msgid "Select all records"
 msgstr "Kies alle rekords"
 
-#: ../flask_admin/templates/bootstrap2/admin/model/list.html:120
-#: ../flask_admin/templates/bootstrap3/admin/model/list.html:119
+#: ../flask_admin/templates/bootstrap4/admin/model/list.html:119
 msgid "Select record"
 msgstr "Kies rekord"
 
-#: ../flask_admin/templates/bootstrap2/admin/model/list.html:185
-#: ../flask_admin/templates/bootstrap3/admin/model/list.html:186
+#: ../flask_admin/templates/bootstrap4/admin/model/list.html:195
 msgid "Please select at least one record."
 msgstr "Kies ten minste een rekord."
 
-#: ../flask_admin/templates/bootstrap2/admin/model/row_actions.html:34
-#: ../flask_admin/templates/bootstrap3/admin/model/row_actions.html:34
-msgid "Are you sure you want to delete this record?"
-msgstr "Is jy seker jy wil hierdie rekord verwyder?"
+#~ msgid "ObjectId equals"
+#~ msgstr ""
```

Comment:
Incidentally, these commented out strings are obsolete; if you no longer need these translations (as opposed to "slightly modified"/"temporarily renamed") you can remove them.

Reply:
Yep I spotted those. I'm happy to leave them - easy to restore later if somehow they come back, and there aren't so many that they're really causing a problem (AFAIK).

Reply:
Actually if they _do_ come back they won't get automatically restored (and if they get obsoleted again even for a single run, the old translations will be removed automatically)… I'd suggest relying on git history instead, like it's done with regular code :sweat_smile:

Reply:
Noted, thanks

Reply:
> Actually if they _do_ come back they won't get automatically restored (and if they get obsoleted again even for a single run, the old translations will be removed automatically)… I'd suggest relying on git history instead, like it's done with regular code 😅

That's really not my experience. If the previously obsolete string reappears in the source code and you run babel extract again, babel extract will:
1. Recognize that the string is present in the source code again.
2. Uncomment the translation and restore it to the active list of translatable strings
3. The translation itself (if already provided) will also be restored, assuming the context hasn't changed (same msgid)